app/src/main/res/drawable/ic_launcher_monochrome.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="108dp"
+    android:height="109dp"
+    android:viewportWidth="108"
+    android:viewportHeight="109">
```

Comment:
Noticed these are 108 and 109 instead of being a perfect square. Is this intentional?

Reply:
Not entirely sure to be honest. This are the proportions when importing the vector asset in AS. I will ask and update this thread accordingly!

Reply:
According to UX the export they have is the same, they said not to worry about it for now.